sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md

```diff
@@ -133,118 +59,33 @@ The authentication events trigger allows a function to be executed when an authe
-        "actions": [
-            {
-                "@odata.type": "microsoft.graph.provideClaimsForToken",
-                "claims": {
-                    "DateOfBirth": "01/01/2000",
-                    "CustomRoles": [
-                            "Writer",
-                            "Editor"
-                        ]
-                    }
-             }
-        ]
-    }
-}
-```
+* Open the project that was created in the prior step. (How to get started)
+* Follow [these](https://learn.microsoft.com/entra/identity-platform/custom-extension-tokenissuancestart-setup?tabs=visual-studio%2Cazure-portal&pivots=nuget-library#build-and-run-the-project-locally) steps to test your app locally.
 
 ## Troubleshooting
 
 * Visual Studio Code
-  * If running in Visual Studio Code, you get an error along the lines of the local Azure Storage Emulator is unavailable, you can start the emulator manually.! (Note: Azure Storage emulator is now deprecated and the suggested replacement is [Azurite](https://learn.microsoft.com/azure/storage/common/storage-use-azurite?tabs=visual-studio))
+  * If running in Visual Studio Code, you get an error along the lines of the local Azure Storage Emulator is unavailable, you can start the emulator manually. (Note: Azure Storage emulator is now deprecated and the suggested replacement is [Azurite](https://learn.microsoft.com/azure/storage/common/storage-use-azurite?tabs=visual-studio))
   * If using Visual Studio Code on Mac please use [Azurite](https://learn.microsoft.com/azure/storage/common/storage-use-azurite?tabs=visual-studio)
-  * If you see the following error on Windows (it's a bug) when trying to run the created projected.
-  * This can be resolved by executing this command in powershell `Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Scope LocalMachine` more info on this can be found [here](https://github.com/Azure/azure-functions-core-tools/issues/1821) and [here](https://learn.microsoft.com/powershell/module/microsoft.powershell.core/about/about_execution_policies?view=powershell-7)
+
+## Publish
+
+* Follow the instruction [here](https://learn.microsoft.com/entra/identity-platform/custom-extension-tokenissuancestart-setup?tabs=visual-studio%2Cazure-portal&pivots=nuget-library#deploy-the-function-and-publish-to-azure) to create and publish your Azure Application.
+* To determine your published posting endpoint, combine the azure function endpoint you created, route to the listener and listener code, the listen code can be found by navigating to your azure function application, selecting "App Keys" and copying the value of AuthenticationEvents_extension.
+  * For example: "https://azureautheventstriggerdemo.azurewebsites.net/runtime/webhooks/AuthenticationEvents?code=(AuthenticationEvents_extension_key)&function=OnTokenIssuanceStart"
+* Make sure your production environment has the correct application settings for token authentication.
 
 ## Next steps
```

Comment:
Move the link to Configure a custom claim provider first. 

The logging, metrics, and azure sdk is secondary

Reply:
Updated